package.json

```diff
@@ -73,11 +73,11 @@
   },
   "devDependencies": {
     "aws-sdk": "^2.3.3",
-    "chai": "^1.10.0",
+    "chai": "^3.5.0",
     "chai-fs": "^0.1.0",
```

Comment:
There is no version of `chai-fs` that would work with the latest version of `chai`